### PR TITLE
Fix circleCI build on 2.2.x

### PR DESCRIPTION
**Pull request by dstansby**
Should help the circleCI build (and maybe even fix it!)

### CODE REVIEW

File: doc/users/prev_whats_new/whats_new_1.0.rst [hunk 1]
```diff
@@ -25,13 +25,6 @@ Jae-Joon Lee has written :mod:`~matplotlib.gridspec`, a new module for
 doing complex subplot layouts, featuring row and column spans and
 more.  See :doc:`/tutorials/intermediate/gridspec` for a tutorial overview.
 
-.. figure:: ../../gallery/userdemo/images/sphx_glr_demo_gridspec01_000.png
```
Comment (dstansby) [inline doc/users/prev_whats_new/whats_new_1.0.rst]: I have no idea why but the build was complaining this file wasn't readable; unless anyone else wants to put the effort in finding out why I'm proposing just removing this snippet altogether.

Comment (timhoffm) [reply doc/users/prev_whats_new/whats_new_1.0.rst]: I would have objections on the master branch, but since 2.x doesn't live forever that's fine.

Comment (tacaswell) [reply doc/users/prev_whats_new/whats_new_1.0.rst]: I suspect what happened here is some changes get backported and some did not.  We will just have to be careful when the 2.x branch gets merged up.

**Comment by dstansby** [comment]
Going to self-merge with one review to un-break CI

**Pull request merged**